pkgs/applications/video/streamlink/default.nix

```diff
@@ -1,21 +1,42 @@
-{ stdenv, pythonPackages, fetchFromGitHub, rtmpdump, ffmpeg_3 }:
+{ lib
+, pythonPackages
+, fetchFromGitHub
+, rtmpdump
+, ffmpeg_3
```

Comment:
Doesn't it build with `ffmpeg` :thinking: ?

Reply:
Only thing I changed was the formatting.